### PR TITLE
Put Nearform logo in bottom left

### PR DESCRIPTION
**Pull request by DylanC**
Issue:
Eyes are drawn to the NearForm logo although it is a secondary element in terms of hierarchy.

Fix:
1. Remove the Nearform logo from the header. https://github.com/nearform/node-clinic-common/pull/24 (Merged)
2. This PR will put the logo in the bottom left. Since https://github.com/nearform/node-clinic-flame/pull/115 created the space required. (Merged)

image

~~Recommended to merge https://github.com/nearform/node-clinic-flame/pull/115 before this one.~~ (Merged)

Position of logo will be adjusted via:
https://github.com/nearform/node-clinic-common/pull/29

### CODE REVIEW

File: visualizer/ui.js [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
+const d3 = require('./d3.js')
+const path = require('path')
 const events = require('events')
 const htmlContentTypes = require('./html-content-types.js')
 const debounce = require('lodash.debounce')
 const DataTree = require('./data-tree.js')
 const History = require('./history.js')
 const spinner = require('@nearform/clinic-common/spinner')
+const logo = require(path.join(__dirname, 'nearform-logo.svg'))
```
Comment (goto-bus-stop) [inline visualizer/ui.js:11]: You can only `require()` Node.js scripts and JSON files. To read files, `fs.readFileSync()` is the appropriate function.

`fs` is a Node.js module that is not available in browsers, and this visualizer code only runs in the browser. But the visualizer is built with browserify, and uses the brfs transform:
https://github.com/nearform/node-clinic-common/blob/7e6ed2f74b08bfc60fd690c4c19e0714a04d80d9/scripts/build-js.js#L18
`brfs` can handle some `fs` functions at build time, so you can do
```js
const fs = require('fs')
const logo = fs.readFileSync(path.join(__dirname, 'nearform-logo.svg'), 'utf8')
```
and brfs will replace that `fs.readFileSync()` call with the contents of the svg as a string. I think with that change, this should be well on its way!

Comment (DylanC) [reply visualizer/ui.js]: @goto-bus-stop - Thanks. The svg is being pulled in now. Just working on the d3 code to append the a tag etc.

**Comment by jasnell** [comment]
Looks a little offset vertically. Can it be centered vertically in the footer bar? Otherwise lgtm

**Comment by DylanC** [comment]
@jasnell - Have a look here:

> Position of logo will be adjusted via:

nearform/node-clinic-common#29

**Pull request merged**